delfin/api/schemas/storage_capabilities_schema.py

```diff
@@ -67,7 +67,7 @@
                                                 'maxLength': 255}
                             },
                         },
-                        'readRequests': {
+                        'readIops': {
                             'type': 'object',
                             'properties': {
                                 'unit': {'type': 'string', 'enum': ["IOPS"]},
```

Comment:
If it's enum, any other values expected instead of IOPS

Reply:
Right, in case of IOPS and % we are not expecting another units.. but we want to follow the same schema for all metrics so that we can pass this to a common json schema validator.